easybuild/easyconfigs/p/python-igraph/python-igraph-0.11.4-foss-2023a.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = 'PythonBundle'
+
+name = 'python-igraph'
+version = '0.11.4'
+
+homepage = 'https://igraph.org/python'
+description = """Python interface to the igraph high performance graph library, primarily aimed at complex network
+ research and analysis."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('pkgconf', '1.9.5'),
+    ('CMake', '3.26.3'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Clang', '16.0.6'),
+    ('libxml2', '2.11.4'),
+    ('zlib', '1.2.13'),
+    ('igraph', '0.10.10'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('pycparser', '2.21', {
+        'checksums': ['e644fdec12f7872f86c58ff790da456218b10f863970249516d60a5eaca77206'],
+    }),
+    ('cffi', '1.16.0', {
```

Comment:
pycparser and `cffi` are included in `Python-bundle-PyPI`, please use that as dependency and remove these 2 extensions